redux/rootReducer.ts

```diff
@@ -16,13 +18,15 @@ type State = {
   skillRatingsState: SkillRatingsState;
   userGoalsState: UserGoalsState;
   profileState: ProfileState;
+  skillifyUsersState: SkillifyUsersState;
```

Comment:
Could this just be userState? Feels more in line with the naming elsewhere in the app

Reply:
Oh kk sounds good.